src/faucet/constants.rs

```diff
@@ -166,6 +166,34 @@ impl FaucetInfo {
             Network::Testnet => 314159, // chainlist.org/chain/314159
         }
     }
+
+    /// Returns the maximum allowed gas limit.
+    #[allow(dead_code)]
+    pub fn max_gas_limit(&self) -> u64 {
+        match self {
+            FaucetInfo::MainnetFIL => 10_000_000,
```

Comment:
Create constants for these magic numbers as well, something like `CALIBNET_USDFC_GAS_LIMIT_WITH_BUFFER`

Reply:
done